CMakeLists.txt

```diff
@@ -93,6 +92,13 @@ endif()
 # SPIRV change starts
 option(ENABLE_SPIRV_CODEGEN "Enables SPIR-V code generation." OFF)
 option(SPIRV_BUILD_TESTS "Build targets for the SPIR-V unit tests." OFF)
+
+# Enable SPIR-V CodeGen for Linux by default.
+if(NOT WIN32)
+  set(ENABLE_SPIRV_CODEGEN ON)
+  set(SPIRV_BUILD_TESTS ON)
```

Comment:
We should probably avoid enabling SPIR-V tests by default. Normally only DXC developers care about tests.

Reply:
I can make this change in a future CL. Thanks